src/textField/index.tsx

```diff
@@ -184,7 +185,7 @@ export default class TextField extends PureComponent <TextFieldProps, TextFieldS
             />
           }
           {
-            !isTextArea &&
+            shouldDisplayButton &&
```

Comment:
Question about the existing: why do we always show a button component - albeit an empty one - on all inputs?

Reply:
That is a good question, that's the way it is right now